locale/foreman.pot

```diff
@@ -14994,38 +15037,69 @@ msgstr ""
 msgid "Review before build"
 msgstr ""
 
+#:
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/BuildModal.js:68
+msgid "Build enables host {hostName} to rebuild on next boot"
+msgstr ""
+
+#:
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/BuildModal.js:78
+msgid "This action will delete this host and all its data (i.e facts, report)"
+msgstr ""
+
 #:
 #: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/BuildModal.js:90
 msgid "No errors detected"
 msgstr ""
 
 #:
-#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:14
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:18
 msgid "Host %s has been removed successfully"
 msgstr ""
 
 #:
-#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:45
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:26
+msgid ""
+"This will delete the VM and its disks. This behavior can be changed via global"
+" setting \"Destroy associated VM on host delete\"."
+msgstr ""
+
+#:
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:29
+msgid ""
+"VM and its disks will not be deleted. This behavior can be changed via global "
+"setting \"Destroy associated VM on host delete\"."
+msgstr ""
+
+#:
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:39
 msgid "Delete host?"
 msgstr ""
 
 #:
-#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:46
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:40
 msgid "Delete host"
 msgstr ""
 
 #:
-#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:74
+#: ../webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:58
+msgid ""
+"Are you sure you want to delete host {host}? This action is irreversible. {cas"
+"cade}"
```

Comment:
what is `{cascade}`? looks like a separate string that should be translated separately and concatenated rather than interpolated into this one.

Reply:
It is part of the string, I agree that it might be better to add it as a separate html object but that can be done in a different pr.
the `cascade` variable is translated separately here:
https://github.com/theforeman/foreman/blob/8791ac63e6203bf6ffef361981f1438824633c8f/locale/foreman.pot#L15061-L15063